.github/workflows/lint.yml

```diff
@@ -0,0 +1,16 @@
+name: Lint
+
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: AutoCorrect
+        uses: huacnlee/autocorrect-action@v2
```

Comment:
Documentation is different from source code, and sometimes it shouldn't strictly adhere to a specific formatting strategy. It requires some flexibility, as mentioned in https://github.com/apache/eventmesh-site/pull/226#discussion_r1645707625. We would like to have control over the formatting strategy.

Additionally, as a formatting check tool, it should support excluding certain paths and provide options to either output warnings only or fail the CI when formatting issues are detected.

Reply:
> provide options to either output warnings only or fail the CI when formatting issues are detected.

May I ask that what is your action's support level for this feature?